mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -252,19 +252,32 @@ func resourceStorageBucket() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				MaxItems: 1,
+				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"main_page_suffix": {
 							Type:         schema.TypeString,
 							Optional:     true,
 							AtLeastOneOf: []string{"website.0.not_found_page", "website.0.main_page_suffix"},
 							Description:  `Behaves as the bucket's directory index where missing objects are treated as potential directories.`,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								if old != "" && new == "" {
+									return true
+								}
+								return false
```

Comment:
```suggestion
								return old != "" && new == ""
```
Missed this earlier, we can reduce the lines by just returning the condition instead of true/false when the condition is true/false

Reply:
Good catch! Thanks